docs/5-design-patterns/hometask/src/shipment-decorator.ts

```diff
@@ -0,0 +1,24 @@
+import { Shipment } from './shipment';
+import { ShipmentState } from './types/types';
+
+export class ShipmentDecorator {
```

Comment:
Please, notice, there is an option to create not one but several decorators for each flag